core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/MetricsServiceITest.java

```diff
@@ -867,7 +867,7 @@ public void testBucketPercentiles() {
 
         NumericBucketPoint bucket = actual.get(0);
 
-        assertEquals(testSize, bucket.getSamples());
+        assertEquals((Integer) testSize, bucket.getSamples());
```

Comment:
Without the cast, it is ambiguous and results in a compiler error. I suppose alternatively I could have changed testSize to be an Integer.